packages/fxa-auth-server/lib/payments/stripe-firestore.ts

```diff
@@ -117,9 +160,9 @@ export class StripeFirestore {
     }
 
     return customerSnap.docs[0].ref
-      .collection('subscriptions')
+      .collection(this.subscriptionCollection)
       .doc(invoice.subscription)
-      .collection('invoices')
+      .collection(this.invoiceCollection)
```

Comment:
So a customer document in Firestore will have multiple sub documents (called collections) here for the subscriptions and invoices for the customer?

Reply:
Their own sub collections, yes.